api/routes/family.py

```diff
@@ -60,14 +67,15 @@ async def import_pedigree(
 
 
 @router.get(
-    '/{project}/pedigree', operation_id='getPedigree', response_class=StreamingResponse
+    '/{project}/pedigree', operation_id='getPedigree',
 )
 async def get_pedigree(
     internal_family_ids: List[int] = Query(None),
+    response_type: ContentType = ContentType.JSON,
     replace_with_participant_external_ids: bool = False,
     replace_with_family_external_ids: bool = False,
```

Comment:
While we're changing this endpoint, could we set these to `True`, I feel like that's the more common use case, and I'm pretty confident that we don't have clients using this endpoint (because it didn't work in the Python API earlier).

```suggestion
    replace_with_participant_external_ids: bool = True,
    replace_with_family_external_ids: bool = True,
```

Reply:
Done!